vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_w8a8_statictensor.py

```diff
@@ -0,0 +1,123 @@
+from typing import Callable, List, Tuple, Union
+
+import torch
+from torch.nn import Parameter
+
+# TODO (varun) : Unify ops and custom ops
+from vllm import _custom_ops as custom_ops
+from vllm._C import ops
+from vllm.model_executor.layers.quantization.compressed_tensors.schemes import (
+    CompressedTensorsScheme)
+from vllm.model_executor.utils import set_weight_attrs
+
+__all__ = ["CompressedTensorsW8A8StaticTensor"]
+
+
+class CompressedTensorsW8A8StaticTensor(CompressedTensorsScheme):
+
+    def __init__(self):
+        pass
+
+    def _quantize(self,
+                  x: torch.Tensor,
+                  scales: torch.Tensor,
+                  logical_widths: List[int],
+                  split_dim: int = 0) -> torch.Tensor:
+
+        x_q = torch.empty_like(x, dtype=torch.int8, device="cuda")
+        x_q_split = x_q.split(logical_widths, dim=split_dim)
+        x_split = x.split(logical_widths, dim=split_dim)
+
+        for q, dq, scale in zip(x_q_split, x_split, scales):
+            ops.static_scaled_int8_quant(q, dq, scale.item())
```

Comment:
This usage of scale may results in some performance issue, because .item() may trigger CPU-GPU sync?